networkit/cpp/distance/test/DistanceGTest.cpp

```diff
@@ -153,22 +153,45 @@ TEST_F(DistanceGTest, testAStar) {
 }
 
 TEST_P(DistanceGTest, testAdamicAdar) {
-    auto G = generateERGraph(500, 0.03);
+    Graph G(100);
```

Comment:
This is a parametric test, you can use:
```suggestion
    Graph G(100, isWeighted(), isDirected());
```